assets/styles/style.css

```diff
@@ -418,6 +429,36 @@ p {
   box-shadow: -5px -5px 5px #e5e5e5, 5px 5px 5px #e5e5e5;
 }
 
+.menu-wrapper .menu .mobile-menu {
+  position: fixed;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 1;
+  color: #fff;
+  display: flex;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
+  flex-direction: column;
+  opacity: 0.9;
+  background-color: #6070ff;
+}
```

Comment:
- [ ] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png)

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/167505808-1a1b7c4b-d2b6-44c0-a6f5-3014ca635c72.png)

Reply:
Hi @youmari 
This feedback has been well received and rectified.